modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/service/test/SegmentsExperienceLocalServiceTest.java

```diff
@@ -237,20 +237,30 @@ public void testDeleteSegmentsExperienceWithMidrangePriority()
 		throws Exception {
 
 		SegmentsExperience segmentsExperience1 =
-			SegmentsTestUtil.addSegmentsExperience(
-				_group.getGroupId(), _classNameId, _classPK);
+			_segmentsExperienceLocalService.addSegmentsExperience(
+				0, _classNameId, _classPK,
```

Comment:
Would it be better to use SegmentsEntryConstants.ID_DEFAULT instad of 0?

Reply:
ID_DEFAULT is not 0, a experience could never be added as SegmentsEntryConstants.ID_DEFAULT

Reply:
If you are reviewing the tests, does that means that the rest of the code has been reviewed and you have not found any more bugs?

Reply:
https://github.com/liferay-tango/liferay-portal/pull/717#discussion_r558280830

Reply:
Awesome!!!.

So we only have the A/B Testing problem. That is really nice. Thanks a lot for the review :).

Reply:
that's it ;) np

so, you don't want me to continue with the rest of the tests's review?

Reply:
No no, review everything :). I just want to know if we are ready to submit a little bit of this changes to master today.

Reply:
just for the record, if you go to `com.liferay.segments.constants.SegmentsEntryConstants` you'see that:

```java
public static final long ID_DEFAULT = 0;
```

Reply:
Ok, reviewing ;)

Reply:
Ahh, you are talking about the segment, right? That is totally right. I don't know why we didn't use it before, but I won't change the way we do it in other test right now.

I will create a ticket so we change all the 0s in tests for the constant.

Good catch